src/components/Guides/ProgressBar.tsx

```diff
@@ -10,20 +11,39 @@ const ProgressBar = (progress: number) => {
     marginRight: '0.3rem'
   };
 
+  const article = {
+    width: '70vw',
+    height: '5px',
+    background: 'white',
+    boxShadow: '-2px 4px 20px rgba(79, 42, 29, 0.15)',
+    borderRadius: '20px',
+    marginRight: '0.3rem'
+  };
+
   const fillStyles = {
     width: `${progress}%`,
     height: '5px',
     background: '#33BF68',
     borderRadius: '20px'
   };
 
-  return (
-    <div>
-      <div style={barStyles}>
-        <div style={fillStyles} />
+  if (page == 'playlist') {
```

Comment:
I don't think we'll need to split between 'playlist' and 'article'! Based on figma they seem to be the same -- if you'd like, you could even move the text part into this component!